validator/known_fragment_names.go

```diff
@@ -0,0 +1,20 @@
+package validator
+
+import (
+	"fmt"
+
+	"github.com/vektah/gqlparser"
+)
+
+func init() {
+	addRule("KnownFragmentNames", func(observers *Events, addError addErrFunc) {
+		observers.OnFragmentSpread(func(walker *Walker, parentDef *gqlparser.Definition, fragmentDef *gqlparser.FragmentDefinition, fragmentSpread *gqlparser.FragmentSpread) {
+			if fragmentDef != nil {
+				return
+			}
+
+			message := fmt.Sprintf(`Unknown fragment "%s".`, fragmentSpread.Name)
```

Comment:
Message works as sprintf, so you can inline these

Reply:
wow, nice. 
I think we should use `Messagef` instead of `Message`.
https://golang.org/cmd/vet/#hdr-Printf_family